src/LaunchDarkly.ClientSdk/ConfigurationBuilder.cs

```diff
@@ -257,7 +257,7 @@ public ConfigurationBuilder Events(IComponentConfigurer<IEventProcessor> eventsC
         /// </para>
         /// <list type="bullet">
         /// <item><description> The first time this happens in the application, the SDK will generate a
-        /// pseudo-random GUID and overwrite the context's <see cref="Context.Key"/> with this string.
+        /// pseudo-random GUID and overwrite the context's <see cref="Context.FullyQualifiedKey"/> with this string.
```

Comment:
This one should be the normal key, right?

Reply:
Done